sdcm/sct_config.py

```diff
@@ -2447,15 +2447,18 @@ def get_version_based_on_conf(self):  # pylint: disable=too-many-locals
 
     def update_argus_with_version(self, scylla_version: str, package_name: str):
         try:
-            package = Package(name=package_name, date='#NO_DATE',
-                              version=scylla_version,
-                              revision_id='#NO_COMMIT_ID',
-                              build_id="#NO_BUILDID")
-            self.log.info("Saving upgraded Scylla version...")
-            test_config = TestConfig()
-            test_config.init_argus_client(params=self, test_id=self.get("reuse_cluster") or self.get("test_id"))
-            test_config.argus_client().submit_packages([package])
-            test_config.argus_client().update_scylla_version(scylla_version)
+            version_regex = re.compile(r'((?P<short>[\w.~]+)(-0.)?(?P<date>[0-9]{8,8})\.(?P<commit>[\d\w]+))([\-\d])*')
```

Comment:
is this regex valid also for all the cases we have (dev, releases, rc versions?) we need testing for that.
Also, how it differs from `SCYLLA_VERSION_GROUPED_RE` in `version_utils.py`?

Reply:
![image](https://github.com/scylladb/scylla-cluster-tests/assets/7761415/5299932d-499e-413b-a82c-7fbd6d82f48a)

Unless I'm missing something

Reply:
Let put the regex into a central place, and make sure it passes via the same tests the other ones are

Reply:
@fruch added a test and moved to `version_utils.py`